Example/String+MiniApp.swift

```diff
@@ -0,0 +1,12 @@
+import Foundation
+
+extension String {
+
+    func isValidUUID() -> Bool {
+        if(self.range(of: #"[0-9a-f]{8}-?[0-9a-f]{4}-?[0-9a-f]{4}-?[0-9a-f]{4}-?[0-9a-f]{12}+$"#,
```

Comment:
You're using Regex because `UUID()` must be passed a string containing hypens? As far as I'm aware, the `UUID` class on Android behaves the same way.

Do we actually need to support the case when the UUID doesn't have hyphens? I think that the UUID's we're using will always have hyphens. It just seems like this Regex adds a lot of complexity for a case that we probably don't need to handle.

Reply:
@corycaywood 
We had a similar approach like Android but that was validating only host app ID.

I added this logic for two purposes, 
1. HostAppId - UUID Type
    - UUID String (with hyphens)
2. Subscription key
    - String with the same length as UUID but just without hyphens

I thought it would be good to have a unique function that tells whether it is valid HostAppID (UUID) and Subscription key.
This function is not exactly meant to validate UUID (w or w/o hyphens) but to validate HostAppId and subscription key

Reply:
Ah, I understand now. But the subscription key isn't a UUID, so I don't think it should be validated in the same way. Actually, I don't think we should do any validation on the subscription key because we can't predict what format it will be. 

Also, this change means that validation for the host app id won't be correct, right? Because the host app id is always supposed to have hyphens.

Reply:
It can validate both.

Ok, I will revert back the change related to the old style that will validate Host app ID (UUID type) and Subscription key (just make sure it is not empty) 👍